forge-gui/res/cardsfolder/upcoming/hierophant_biotitan.txt

```diff
@@ -0,0 +1,14 @@
+Name:Hierophant Bio-Titan
+ManaCost:10 G G
+Types:Creature Tyranid
+PT:12/12
+K:Vigilance
+K:Reach
+K:Ward:2
+A:SP$ PermanentCreature | Cost$ 10 G G RemoveAnyCounter<X/P1P1/Creature.YouCtrl/among creatures you control> | CostDesc$  As an additional cost to cast this spell, you may remove any number of +1/+1 counters from among creatures you control. 
```

Comment:
@Northmoc 
can you check if there's a way to include the flavor words nicely here?
Currently it seems to ignore the CostDesc$ and auto builds it, so it will be doubled